config/routes.rb

```diff
@@ -26,7 +26,25 @@
   end
 
   resources :partner_requests, only: [:new, :create, :show, :index]
-  resources :family_requests, only: [:new, :create]
+  resources :family_requests, only: [:new, :create] do
+    resource :pickup_sheets, only: :show
+  end
+
+  post "/child_item_requests_toggle_picked_up/:id", action: :toggle_picked_up,
```

Comment:
How do you feel about making these routes a little more restful? Ill put my comments inline about what controller names/actions make sense to me.